src/MycilaESPConnect.h

```diff
@@ -267,5 +267,8 @@ namespace Mycila {
 
     private:
       static int8_t _wifiSignalQuality(int32_t rssi);
+      bool _restartPending = false;
```

Comment:
you do not need this. You can only have 1 field which is called `restartRequestTime`. If non 0, then you need to restart after a delay.